AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -112,6 +112,7 @@ techadvisor.co.uk,forum.notebookreview.com,computerworld.com,skysports.com#$#bod
 !
 !
 !
+qvcuk.com##div.footer-privacy-banner
```

Comment:
Please add this domain to the existing rule. ( `qvc.de##.footer-privacy-banner` )

Reply:
Thank you

Reply:
@Sergey-Lyapin requested you to remove `qvcuk.com##div.footer-privacy-banner` line and instead add domain to the existing rule `qvc.de##.footer-privacy-banner`, the same as what I requested in https://github.com/AdguardTeam/AdguardFilters/pull/102372 and https://github.com/AdguardTeam/AdguardFilters/pull/102373

Reply:
Hm, I didn't notice your PR and already fixed this issue, sorry. 
Please take a look how it should be:
https://github.com/AdguardTeam/AdguardFilters/commit/d1e72522aefdb49ded7dafaa3fd4f7208c1c3fcc